scripts/update-dependencies/Config.cs

```diff
@@ -33,7 +33,7 @@ public class Config
         private Lazy<string> _email = new Lazy<string>(() => GetEnvironmentVariable("GITHUB_EMAIL"));
         private Lazy<string> _password = new Lazy<string>(() => GetEnvironmentVariable("GITHUB_PASSWORD"));
         
-        private Lazy<string> _coreFxVersionUrl = new Lazy<string>(() => GetEnvironmentVariable("COREFX_VERSION_URL", "https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/corefx/master/Latest.txt"));
+        private Lazy<string> _coreFxVersionUrl = new Lazy<string>(() => GetEnvironmentVariable("COREFX_VERSION_URL", "https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/corefx/release/1.0.0/Latest_Packages.txt"));
```

Comment:
It is read from an EnvironmentVariable.  If that EnvironmentVariable doesn't exist, the hard-coded string is just a default.  Putting the default in scripts or in a config file doesn't really make a difference to me.  It just seems easier to keep all the defaults in the same code file.